src/index.css

```diff
@@ -1,13 +1,25 @@
+
+
 body {
+  background-attachment: fixed;
+  overflow-x: visible;
+  overflow-y: hidden;
+  background-image:url("http://pharmacie-saint-james.fr/wp-content/uploads/2016/03/Lumi%C3%A8re-divine.jpg");
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-size: 30px;
+  text-align: center;
+  font-family: 'Oswald', sans-serif;
+  color: #fe0000;
 }
+/* TITRE DE FILM */
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
-}
+.trailertitle{
+  letter-spacing: 0.2em;
+  text-align: center;
+  margin-bottom: 3%;
+  margin-top: 3%; 
+ }
```

Comment:
It should not be put as a global global style inside the index.css but in the correct component.